experimenter/tests/integration/nimbus/test_demo_app_integration.py

```diff
@@ -57,25 +75,19 @@ def test_create_new_rollout_approve_remote_settings_demo_app(
     assert True in [experiment_name in item.text for item in home.tables[0].experiments]
 
     # demo app frontend, default displays "Not Enrolled"
-    navigate_to_demo_app_frontend(selenium)
-    result_text_element = WebDriverWait(selenium, 10).until(
-        EC.presence_of_element_located(
-            (By.XPATH, "//h1[contains(text(), 'Not Enrolled')]")
-        )
-    )
+    demo_app = DemoAppPage(driver)
+
+    demo_app.navigate_to()
+    result_text_element = demo_app.wait_for_result_text(["Not Enrolled"])
```

Comment:
Once you move this Page Model to the ```pages``` folder, you'll just have to import it like our other tests.